master/docs/relnotes/index.rst

```diff
@@ -72,6 +72,8 @@ Deprecations, Removals, and Non-Compatible Changes
 
 * Master/worker protocol has been changed:
 
+  * ``slave_commands`` key in worker information was renamed to ``worker_commands``.
```

Comment:
Ah, yes. Let's merge relnotes changes as they already distributed in PRs, I'll resolve/rebase/update them when conflicts will arise.